doc/nrf/protocols/bt/bt_qualification/index.rst

```diff
@@ -20,10 +20,12 @@ Pre-qualified design
 ********************
 
 A *design* is a specific configuration of hardware or software implementation (or both) of adopted Bluetooth specifications, either in parts or as a whole.
-If relevant, you can use an existing design that has already been qualified (called *pre-qualified design*).
+If relevant, you can use an existing design that has already been qualified (called *pre-qualified design*) and has a Qualification Design ID (QDID) number.
 This allows you to speed up your product's qualification process by omitting a testing phase.
 
 The |NCS| contains qualified portions of various Bluetooth features (for example, :ref:`ug_ble_controller_softdevice` versions).
+If your product uses a qualified Bluetooth stack provided as part of the |NCS|, you can inherit this qualification from Nordic Semiconductor, provided that you do not introduce any changes to the stack.
+To find the QDID, go to the Compatibility Matrix relevant for the SoC or SiP model you use, and open the Bluetooth QDID section.
```

Comment:
It will be really nice to link "Compatibility Matrix" page for 52840 here.
```suggestion
To find the QDID, go to the Compatibility Matrix relevant for the SoC or SiP model you use, and open the Bluetooth QDID section. For example Compatibility Matrix for nRF52840 SoC is available here<link>.
```

Reply:
Used a bit of a different wording.